src/naquibalam/A13_StringFunction.java

```diff
@@ -0,0 +1,62 @@
+package naquibalam;
+
+import java.util.Scanner;
+
+public class A13_StringFunction {
+	
+	void getAllDuplicateChar() {
+		Scanner scanner1 = new Scanner(System.in);
```

Comment:
always write parameterized methods, reason is we always try to create our program/code in such way that if someone wants to call our method on their inputs then still it works. hope you get my point.